samples/cypress/dashboards/chart-interaction-selection/demo.mjs

```diff
@@ -1,23 +1,25 @@
-import CSVConnector from '../../../../code/es-modules/Data/Connectors/CSVConnector.js';
-import Board from  '../../../../code/es-modules/Dashboards/Board.js';
 import PluginHandler from  '../../../../code/es-modules/Dashboards/PluginHandler.js';
+import Dashboards from '../../../../code/es-modules/masters/dashboards.src.js';
 import Highcharts from  '../../../../code/es-modules/masters/highcharts.src.js';
 import HighchartsPlugin from  '../../../../code/es-modules/Extensions/DashboardPlugins/HighchartsPlugin.js';
 
 HighchartsPlugin.custom.connectHighcharts(Highcharts);
 PluginHandler.addPlugin(HighchartsPlugin);
 
-// A shared connector
-const connector = new CSVConnector({
-    csv: `$GME,$AMC,$NOK
- 4,5,6
- 1,5,2
- 41,23,2`,
-    firstRowAsNames: true
-});
-connector.load();
-
-const board = new Board('container', {
+const board = Dashboards.board('container', {
+    dataPool: {
+        connectors: [{
+            name: 'connector-1',
```

Comment:
I would prefer to use the id instead of the name. We use id's in cells/rows and other elements to assign themselves.

Reply:
This will be covered in a separate PR.